src/validateFilenames.ts

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+/* tslint:disable */
```

Comment:
Is `/* tslint:disable */` needed here? I think this disable TSLint for the entire file.

Reply:
There's a bug in `tslint` ignore where if the file begins with `#!/usr/bin/env node`, `/* tslint:disable */` at the root scope of the file doesn't apply correctly! I was maniacally trying different things to ignore the first `require('babel-polyfill');`.

So `/* tslint:disable */` is actually not doing anything, but that's not why I left it there. I left it there because I  forgot! 😅